### ISSUE TITLE
streamzz  info stream, instead of the media stream

### ISSUE THREAD

**Issue by kolekan** [jsergio123/script.module.resolveurl#628]
Hi, i just found that streamzz.to link plays an info stream instead of the media stream. But if i play any streamzz link in chrome once, then for 2 maybe 3 hours ,  streamzz links plays the correct streams.

**Comment by Gujal00** [comment]
Yes looks like they have added a invisible captcha to whitelist your ip for a couple of hours. So yes if you use from browser then it will work from ResolveURL for a few hours. Need to figure out how to solve the captcha, seems complex obfuscated js code
```https://streamzz.to/cdn-cgi/challenge-platform/h/b/scripts/invisible.js```
